src/scss/components/_form-control.scss

```diff
@@ -42,6 +42,21 @@
     display: block;
   }
 
+  &--no-asterisks {
+    .required-field {
+      // Override the asterisk
+      .#{prefix(label)}:after {
+        content: "" !important;
+      }
+    }
+
+    .optional-field {
+      .#{prefix(label)}:after {
+        content: "(optioneel)";
```

Comment:
I'd rather see this handled in the label template so we can properly apply translations - if we support English localization (which we already sort of do), then this will look out of place.

Reply:
Yes, this was lazy :stuck_out_tongue: